java/manager-build.xml

```diff
@@ -632,4 +632,16 @@
     </exec>
 
   </target>
-</project>
+
+  <target name="make-eclipse-project"
+          description="Configures this checkout as an eclipse project.">
+    <copy file="${rhn-home}/conf/eclipse/.project-template" tofile="${rhn-home}/.project" overwrite="false" />
+    <copy toDir="${rhn-home}" overwrite="false">
+      <fileset dir="${rhn-home}/conf/eclipse/">
+        <include name=".checkstyle" />
+        <include name=".classpath" />
+        <include name=".settings/*" />
+        <!-- include name=".externalToolBuilders/*" /-->
+      </fileset>
+    </copy>
+  </target>
```

Comment:
one of the checks complained  too
```suggestion
  </target>
</project>
```

Reply:
doh! I got trick by `git rebase` two times... will be good now